File: Java/Easy/242/242.java
class Solution {
    public static boolean isAnagram(String s, String t) {

        char sNew[] = s.toCharArray();
        char tNew[] = t.toCharArray();

        if (sNew.length != tNew.length) {
            return false;
        }
        Arrays.sort(tNew);
        Arrays.sort(sNew);


        return Arrays.equals(sNew, tNew);
    }
}

File: Go/Easy/242/242.go
import (
    "fmt"
    "sort"
)

func isAnagram(s string, t string) bool {

    sNew := sortString(s)
    tNew := sortString(t)

    if sNew == tNew {
        return true
    }
    return false
}

func sortString(s string) string {

	runes := []rune(s)
	sort.Slice(runes, func(i, j int) bool {
		return runes[i] < runes[j]
	})
	return string(runes)
}
